add-on/chrome/bootstrap.js

```diff
@@ -1039,6 +1039,14 @@ function loadDefaultPrefs() {
       }
     }
   });
+
+  if (Services.vc.compare(Services.appinfo.version, "47.0a1") < 0) {
+    branch.setBoolPref("loop.remote.autostart", false);
```

Comment:
True, via the experiment. But that's only for the beta audience. e10s will *not* be turned on in 46 GA.